CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+# CMakeLists files in this project can
+# refer to the root source directory of the project as ${HRPD_SOURCE_DIR} and
+# to the root binary directory of the project as ${HRPD_BINARY_DIR}
+cmake_minimum_required(VERSION 3.22.1)
```

Comment:
I tested successfully with 3.19.4, so you can set the minimum to that.

Reply:
Done